build/forklift-cli-download/entrypoint.sh

```
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+kubectl-mtv mcp-server --sse \
+  --host 127.0.0.1 --port 8081 \
+  --output-format "${MCP_OUTPUT_FORMAT:-markdown}" \
+  ${MCP_MAX_RESPONSE_CHARS:+--max-response-chars "$MCP_MAX_RESPONSE_CHARS"} \
+  ${MCP_KUBE_SERVER:+--server "$MCP_KUBE_SERVER"} \
+  ${MCP_KUBE_TOKEN:+--token "$MCP_KUBE_TOKEN"} \
+  $([ "${MCP_KUBE_INSECURE}" = "true" ] && echo --insecure-skip-tls-verify) \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)